sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/pypi.yaml

```diff
@@ -0,0 +1,44 @@
+provider:
+  metadata:
+    name: PyPI API tokens should not be disclosed
```

Comment:
I think you got confused with this change. This is supposed to contain the name of the provider (which it previously did).

Reply:
My bad. What is `name` used for in the rule? Because I've seen it contain both the [secret](https://github.com/SonarSource/sonar-text/blob/master/sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/alchemy.yaml#L11) and [the name of the rule](https://github.com/SonarSource/sonar-text/blob/master/sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/airtable.yaml#L16) interchangeably (not in `provider` though, so I'll change this ASAP).

Reply:
In my mind, until now, it was containing the name of the rule. That said, I don't think this is a big deal, this information is not used afterward to the best of my knowledge.